docs/topics/mta-web-administrator-view.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * docs/getting-started-guide/master.adoc
+
+:_content-type: CONCEPT
+[id="mta-web-administration-view_{context}"]
+= User Interface views
+
+The {ProductName} ({ProductShortName}) {WebName} and has two user interfaces with different functionalities:
+
+* Administrator view
+* Developer view
+
+== Administrator view
+The administrator view is used by administrators of {ProductShortName} {WebName} to manage the configuration of an instance.
+//![](/Tackle2/Views/AdminMenu.png)
+
+=== Credentials
+The credentials management module enables administrators to create and manage credentials to access private repositories.
+//![](/Tackle2/Views/CredMain.png)
+
+The credentials page displays the available credentials, *Edit* and *Delete* buttons and the following fields:
+
+* *Name*
+* *Description*
+* *Type*
+** *Source Control*
+** *Username/Password*
+** *Source Private Key/Passphrase*
+** *Maven Settings File*
+** *Proxy*
+* *Type specific information*
+* *Created by*
+
+NOTE: Credential information entered, such as keys and passphrases, is hidden or displayed as [Encrypted], except for Maven Settings files.  When Maven Setting files are created, the file is uploaded and its contents are displayed in the text box to enable the user to make changes if desired. [Encrypted] only appears on edit for Maven Settings files.
+
```

Comment:
Please re-check and update this note.
 Now, The user can also see and change the contents of keys and passphrases
![Screenshot from 2022-11-03 21-15-07](https://user-images.githubusercontent.com/87791967/199768679-5bcf722a-5180-4371-8819-69037da7185d.png)

Reply:
@rromannissen please see above. How should I edit this note based on this information?

Reply:
@rromannissen Should I remove the note?

Reply:
I don't think this is accurate, the example here is for the SCM Private Key file, not for the Maven settings.xml file. I think this should remain as it is right now